packages/qwik/src/optimizer/src/plugins/vite-error.ts

```diff
@@ -0,0 +1,210 @@
+}
+
+vite-error-overlay::part(tip):before {
+  content: "Not sure how to solve this? Visit https://qwik.builder.io or connect with the community on Discord.";
```

Comment:
just an idea. please let me know what u think about it...

Reply:
yeah if this works! i think it's ok!

Reply:
it does but w/o a link 😄 and can't style the url